restock.lic

```diff
@@ -37,7 +37,9 @@ class Restock
       item['buy_num'] = buy_num
       items_to_restock.push(item)
     end
-    get_money_from_specific_bank?("#{coin_needed} copper #{town_currency(town)}", town) if coin_needed > 0
+    ### The following line is for the bribe to the night attendant to let you in.
+    coin_needed += 2502 if UserVars.sun['night'] = true && town == /\bShard\b/
```

Comment:
I think there's a typo in `UserVars.sun['night'] = true` as that's assigning the value `true` to the variable rather than checking if it's nighttime. I think you can just remove the `= true` and ruby will evaluate to true if the expression is truthy. To be super explicit, then need a second equal sign for comparison.
```suggestion
    coin_needed += 2502 if UserVars.sun['night'] && town == /\bShard\b/
```

Reply:
Changed to ` coin_needed += 2502 if UserVars.sun['night'] && town == /\bShard\b/`